src/main/nav_create_simulated_image.py

```
@@ -1682,45 +2253,59 @@ def _update_display(self) -> None:
 
     # ---- Selection / drag-move ----
     def _select_model_at(self, img_v: float, img_u: float) -> None:
-        # Check bodies first: order_near_to_far; pick first whose mask contains pixel
+        """Select the topmost model at the given image coordinates based on range ordering.
+
+        Checks bodies and rings together, sorted by range (near to far), and selects
+        the first match (topmost object).
+        """
+        height = int(self.sim_params['size_v'])
+        width = int(self.sim_params['size_u'])
+        v_i = int(round(img_v))
```

Ruff (0.14.11)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)